pkg/binding/convert/postgresql_converter_test.go

```diff
@@ -64,13 +64,32 @@ func TestPostgreSQLConnectionStringConverter_Convert(t *testing.T) {
 						"username": "a-db-user",
 						"password": "password'",
 						"srv":      "true",
-						"options":  "some-db-options",
+						"options":  "some-db-options_that_is_invalid",
 						"database": "local-db",
 					},
 				},
 			},
 			want: "host=example.com:10011 user=a-db-user password=password\\' dbname=local-db",
 		},
+		{
+			name: "Invalid_or_incomplete options are ignored",
+			args: args{
+				binding: fileconfig.ServiceBinding{
+					Name:        "local",
+					BindingType: "postgresql",
+					Provider:    "Crunchy Bridges",
+					Properties: map[string]string{
+						"host":     "example.com:10011",
+						"username": "a-db-user",
+						"password": "password'",
+						"srv":      "true",
+						"options":  "option1=value1&option2",
```

Comment:
I think some option may support empty string?

Reply:
I think empty string should be OK as it shouldn't generate any key/value for the option, but I'll add a test for it. Thanks for pointing it out.

Reply:
I meant to say a option could be set to empty like: "opt=" so that postgresql can use its default.
Some options behavior like that: https://www.postgresql.org/docs/13/libpq-connect.html